.github/workflows/ci.yml

```diff
@@ -13,7 +13,4 @@ jobs:
         run: pip install flake8
       
       - name: flake8 all repo only show erros
-        run: flake8 . || true # preventing CI failing while flake8 issues exist
-
-      - name: flake8 validating correct file(s)
-        run: flake8 asv/main.py asv/repo.py asv/results.py asv/runner.py asv/statistics.py asv/step_detect.py asv/util.py test/benchmark/ asv/__init__.py asv/benchmarks.py asv/config.py asv/benchmark.py
+        run: flake8 .  # preventing CI failing while flake8 issues exist
```

Comment:
You can remove the comment too, as we're not preventing the CI from failing anymore now.